fastlane/Fastfile

```diff
@@ -77,26 +105,32 @@ lane :update_hybrid_common_versions do |options|
     UI.user_error!("Missing `version` argument")
   end
 
-  previous_version_number = parse_previous_hybrid_version
+  current_phc_version = get_phc_version
 
-  # Add to options so if we're chaining lanes together, the other lanes get it
-  options[:previous_hybrid_version] = previous_version_number
-
-  UI.message("ℹ️  Current Purchases Hybrid Common version: #{previous_version_number}")
+  UI.message("ℹ️  Current Purchases Hybrid Common version: #{current_phc_version}")
   UI.message("ℹ️  Setting Purchases Hybrid Common version: #{new_version_number}")
   files_to_update = [
     'plugin.xml'
   ]
 
   if dry_run
-    UI.message("ℹ️  Nothing more to do, dry_run: true")
-  else
-    replace_text_in_files(
-      previous_text: previous_version_number,
-      new_text: new_version_number,
-      paths_of_files_to_update: files_to_update
-    )
+    UI.message('ℹ️  Nothing more to do, dry_run: true')
+    next
   end
+
+  bump_phc_version(
+    repo_name: repo_name,
+    files_to_update: files_to_update,
+    current_version: current_phc_version,
+    next_version: new_version_number,
+    open_pr: options[:open_pr] || false,
+    automatic_release: options[:automatic_release] || false
+  )
+end
+
+desc "Update Hybrid Common and builds examples. Calls update_hybrid_common, then build_hybrid_example and pushes changes to a new branch if open_pr option is true"
+lane :update_hybrid_common do |options|
+  update_hybrid_common_versions(options)
```

Comment:
Should we just rename the `update_hybrid_common_versions` to `update_hybrid_common`?

Reply:
oh yes, I did that in react native, but not here